Translation corrections (#136)
07th-mod/higurashi-console-arcs: Update/some_012.txt

//rおかーさんの手をつかんで止めていたのは、…おとーさんだった。kそして、
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "おかーさんの手をつかんで止めていたのは、…おとーさんだった。",

> 			NULL, "It was Dad... grabbing Mom's hand stopping her.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "It was Dad... grabbing Mom's hand to stop her.", Line_WaitForInput);
```

			NULL, "\"Wh...?!", Line_WaitForInput);
	ModPlayVoiceLS(4, 0, "ps3/s04/00/310600169", 256, TRUE);
	OutputLine(NULL, "　けっ、警察ッ？！」",

> 			NULL, " T-the police?!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Th-the police?!\"", Line_Normal);
```

	ClearMessage();

//rおかーさんは差し出された手を乱暴に振り払うと、拳を握り締めて怒りに震える。けど、おとーさんは気丈にもそれに怯むことなく、…吹っ切れたような口調で静かに言葉を継いでいった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "おかーさんは差し出された手を乱暴に振り払うと、拳を握り締めて怒りに震える。けど、おとーさんは気丈にもそれに怯むことなく、…吹っ切れたような口調で静かに言葉を継いでいった。",

> 			NULL, "Mom violently shook away the hand that was holding her, then clenched her fist and trembled with anger. However, Dad stood firm and showed no fear... He calmly continued in a resolved tone.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Mom violently shook away the hand that was holding her, then clenched her fist and trembled with anger. However, Dad stood firm and showed no fear... He calmly continued in a resolute tone.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>夏美の父</color>", NULL, "<color=#f5e6d3>Natsumi's dad</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s04/00/310700044", 256, TRUE);
	OutputLine(NULL, "「お父さんも、ずっと考えていたんだよ。",
			NULL, "\"I've been thinking for a long time.", Line_WaitForInput);
	ModPlayVoiceLS(4, 0, "ps3/s04/00/310700045", 256, TRUE);
	OutputLine(NULL, "どうしたらいいか。",
			NULL, " About what I should do.", Line_WaitForInput);
	ModPlayVoiceLS(4, 0, "ps3/s04/00/310700046", 256, TRUE);
	OutputLine(NULL, "…元通りに家族揃って幸せに暮らすにはどうしたらいいのか…、ずっと考えていたんだ」",

> 			NULL, " ...I was constantly thinking... about we could live happily as a family like we used to.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " ...I was constantly thinking... about how we could live happily as a family like we used to.\"", Line_Normal);
```
